File: namsor/python/src/CastegroupIndianHinduExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.IndianApi(api_client).castegroup_indian_hindu(
            sub_division_iso31662="IN-UP",
            first_name="Akash",
            last_name="Sharma",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling IndianApi#castegroup_indian_hindu: %s\n" % e)

File: namsor/ruby/src/CastegroupIndianHinduExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::IndianApi.new.castegroup_indian_hindu(
        "IN-UP", # sub_division_iso31662
        "Akash", # first_name
        "Sharma", # last_name
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling IndianApi#castegroup_indian_hindu: #{e}"
end
